app/templates/public/index.hbs

```diff
@@ -12,4 +12,10 @@
       {{public/speaker-list speakers=nonFeaturedSpeakers}}
     </div>
   </div>
+  <br>
+  <div class="sponsors">
+    <div class="row">
+      {{public/sponsor-list sponsors=sponsorsByCategory}}
```

Comment:
No. I think you misunderstood. Components cannot have controllers. Routes have controllers. 

You can directly put the same code you used in the index controller inside the component js file.